app/storage/questionnaire_storage.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from app.data_model.database import QuestionnaireState, commit_or_rollback
+from app.data_model.database import db_session
+
+logger = logging.getLogger(__name__)
+
+
+class QuestionnaireStorage:
```

Comment:
I suppose given that we have QuestionnaireStore holding the data then this name is consistent.